### PR TITLE
Add Kiali deprecation and link to example

### PR DESCRIPTION
**Pull request by chrkl**
**Description**

Changes proposed in this pull request:

- Add deprecation warning for Kiali
- Add link to Kiali example

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/15410

### CODE REVIEW

**Comment by a-thaler** [comment]
In docs/01-overview/main-areas/assets/kyma-areas.svg kiali seems to be mentioned, please remove the logo from the diagram

**Comment by a-thaler** [comment]
On these pages Kiali gets mentioned as well. Please remove or deprecate or link to tutorial

- docs/01-overview/main-areas/observability/README.md
- docs/01-overview/main-areas/service-mesh/smsh-02-default-istio-setup-in-kyma.md
- docs/01-overview/main-areas/service-mesh/smsh-03-istio-sidecars-in-kyma.md
- docs/02-get-started/05-observability.md
- docs/05-technical-reference/00-configuration-parameters/obsv-01-configpara-observability.md

File: docs/01-overview/main-areas/service-mesh/smsh-03-istio-sidecars-in-kyma.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Another security benefit of having a sidecar proxy is that you can perform [requ
 
 Furthermore, Istio proxies improve tracing: Istio performs global tracing and forwards the data to [Kyma's tracing component](../../../01-overview/main-areas/observability/obsv-03-tracing-in-kyma.md) using the [Zipkin protocol](https://zipkin.io). Learn more about the process in [Tracing Architecture](../../../05-technical-reference/00-architecture/obsv-03-architecture-tracing.md).
 
-Kiali is another tool that comes as [separate Kyma component](../../../05-technical-reference/00-architecture/obsv-04-architecture-kiali.md); and Kyma configures Istio to export metrics necessary to support Kiali features that facilitate managing, visualizing, and troubleshooting your service mesh.
+Kiali is another tool that to monitor the service mesh; and Kyma configures Istio to export metrics necessary to support Kiali features that facilitate managing, visualizing, and troubleshooting your service mesh. Learn more about deploying Kiali to your Kyma cluster in our [Kiali example](https://github.com/kyma-project/examples/tree/main/kiali).
```
Comment (a-thaler) [inline docs/01-overview/main-areas/service-mesh/smsh-03-istio-sidecars-in-kyma.md:23]:
```suggestion
Kiali is another tool to monitor the service mesh; and Kyma configures Istio to export metrics necessary to support Kiali features that facilitate managing, visualizing, and troubleshooting your service mesh. Learn more about deploying Kiali to your Kyma cluster in our [Kiali example](https://github.com/kyma-project/examples/tree/main/kiali).
```

**Review by NHingerl — APPROVED** [review]

**Review by lindnerby — APPROVED** [review]

**Pull request merged**